<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myCam</class>
 <widget class="QWidget" name="myCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myCam</string>
  </property>
  <widget class="QLabel" name="label_show">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>300</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CamOpen">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CamClose">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_SaveImg">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
